public/data/food/products.json

```diff
@@ -0,0 +1,248 @@
+[ { "name": "Carrot cake (643g)",
+    "query": {
+      "ingredients": [
+        {
+          "id": "egg",
+          "mass": 120
+        },
+        {
+          "id": "wheat",
+          "mass": 140
+        },
+        {
+          "id": "milk",
+          "mass": 60
+        },
+        {
+          "id": "carrot",
+          "mass": 225
+        }
+      ],
+      "transform": {
+        "code": "AGRIBALU000000003103966",
+        "mass": 545
+      },
+      "packaging": [
+        {
+          "code": "AGRIBALU000000003104019",
+          "mass": 105
+        }
+      ],
+      "distribution": "ambient",
+      "preparation": [
+        "refrigeration"
+      ]
+    },
+    "category": "Pâtisserie" }
```

Comment:
How about formatting this JSON file using the same format as other JSON data files (with prettier)? Also related: do you use the git pre-commit hook checking for JSON formatting? If so, why didn't it prevent you to commit this file?

Reply:
I would also put the `category` property just after the `name` one, so we start each object with two short lines and a big `query` blob, which may makes things slightly more legible (here the `category` field is a bit hidden).